includes/Extension/SearchReplace/class-replace.php

```diff
@@ -71,6 +74,14 @@ public function maybe_enqueue_assets( $hook ) {
 				),
 			)
 		);
+
+		/**
+		 * Instawp installation event handle script 
+		 * */
+		wp_enqueue_script( 'string-locator-instawp', trailingslashit( STRING_LOCATOR_PLUGIN_URL ) . 'build/string-locator-instawp.js', array( 'jquery', 'updates' ), false, 'all' );
```

Comment:
A version string is missing to help avoid cached content when new releases of the plugin are deployed.

The final argument has the `all` keyword, which is not a valid entry,. this is a boolean entry to determine if the script should be enqueued in the footer of the site or not.

Reply:
Reolved in latest commit.